src/init.c

```diff
@@ -37,6 +39,24 @@ extern void _PG_fini(void);
 void
 _PG_init(void)
 {
+	if (!process_shared_preload_libraries_in_progress)
+	{
+	
+		char	   *force_load = GetConfigOptionByName("timescaledb.force_load", NULL, true);
```

Comment:
Confusing config option name. Sounds like this forces the extension to be loaded, while in reality it just forces a check for loaded lib. Maybe better name is `require_preload` or `enforce_preload`.